corehq/apps/reports/templates/reports/filters/multi_option.html

```diff
@@ -28,7 +28,9 @@
         });
     });
     {% if not endpoint %}
-        $('#{{ css_id }}').select2();
+        _.delay(function() {
+            $('#{{ css_id }}').select2();
```

Comment:
When does this bit of JS get executed? Is it before or after DOM ready or unknown?